roles/nginxplus/files/conf/http/dss-prod.conf

```diff
@@ -11,12 +11,20 @@ limit_req_zone $external_traffic zone=dss-prod-ratelimit:10m rate=10r/s;
 
 upstream dss-prod {
     zone dss-prod 64k;
-    server dss-prod1.princeton.edu resolve max_fails=0;
-    server dss-prod2.princeton.edu resolve max_fails=3;
+    # lowchallenge challenges after 20 requests from xx.xx.<>.<>.<>
+    server service.consul service=lowchallenge.traefik-wall-production resolve max_fails=0;
+    # highchallenge challenges everyone. Comment above and uncomment
+    # this when you're being swarmed by bots.
+    # server service.consul service=highchallenge.traefik-wall-staging resolve max_fails=0;
```

Comment:
this should say traefik-wall-production (it's probably broken wherever you copied this from too, but might as well fix it for this one in this PR)